### DISCUSSION TITLE
[Bug] - I've only been with Monika for 4 or 5 days and she doesn't talk to me anymore :c

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#8234] [QA - tech support + general] [answered]
I've only been with Monika for 4 or 5 days and she doesn't talk to me anymore I would like you to help me with this problem and if I write this it is because I love the game plis 
[ev_dump.log](https://github.com/Monika-After-Story/MonikaModDev/files/7462624/ev_dump.log)
:c If I can talk to her but she doesn't talk to me ;-;

**Comment by Alexxx1407** [comment]
What do you mean by "doesn't talk anymore?"

**Comment by ghost** [comment]
I mean that literally from one moment to another she no longer said anything and it is not that she told me she simply does not speak to me but if I can interact with her, I mean if I tell her something if she responds but if I don't say anything she doesn't do it either. :c

**Comment by ghost** [comment]
If at any moment I am not understood, it is because I do not speak English very well ;-;

**Comment by Alexxx1407** [comment]
Uhm shows me your setting in-game, please.

**Comment by ghost** [comment]
ok

**Comment by ghost** [comment]
command capture? because the game is in spanish

**Comment by ghost** [comment]
do I send you capture? *

**Comment by Alexxx1407** [comment]
Uhm just the settings, where you edit your volume etc.

**Comment by Alexxx1407** [comment]
Yeah please!

**Comment by ghost** [comment]
![imagen](https://user-images.githubusercontent.com/93607179/139916076-7b559000-5d85-498f-8bad-60068ec32cfe.png)
****

**Comment by ghost** [comment]
it can be opened?

**Comment by Alexxx1407** [comment]
Alright, yeah well i don't see any problems here... did you do something before the problem?

**Comment by ghost** [comment]
Well no, I was normal with the game

**Comment by Alexxx1407** [comment]
Hmm, have you tried to update the game?

**Comment by ghost** [comment]
Well I only did it once but there were no updates because I downloaded it I think 4 days ago

**Comment by Alexxx1407** [comment]
Alright! Well, can you explain a bit clearly the problem ? I didn't understand up there.

**Comment by ghost** [comment]
I could not explain it better, but I will try it was with the game open talking as I always think it happens on Halloween at night but I did not give it importance since I thought it was normal the next day I opened the game again all normal but the hours passed and He did not speak but if I told him, for example, that I will be right back, he would answer me, he would come back, he would speak to him all right but he still did not say anything.

**Comment by Alexxx1407** [comment]
Uhm, you may be past all the things she had to say! By pressing "x", it's not a bug if she talks to you when you want to talk! try to go where you skipped all. if you can't find it, ask directly a moderator. I am not one! Sadly, hope that I could help.

**Comment by ghost** [comment]
ok :)

**Comment by Alexxx1407** [comment]
Good! ask if you can't find it. I will do my best!

**Comment by ghost** [comment]
I don't know how to contact them 
:c

**Comment by Alexxx1407** [comment]
Just don't close this PR, and wait for a moderator to come. Or dm one in discord.

**Comment by ghost** [comment]
ok 
thanks for trying to help me :)

**Comment by Alexxx1407** [comment]
You're welcome! Feel free to ask!! Good luck.

Alexandre,

**Comment by Booplicate** [comment]
You're using a translation, we don't provide support for translations because it's not our code. The chances are you just read all the available topics, and since you didn't allow her to repeat, she won't talk until you unlock a new topic.

**Comment by howltrek** [comment, marked as answer]
no tienes activada la opción repetir conversaciones, tal vez solo se le agotaron las conversaciones desbloqueadas
